pkg/virt-controller/watch/migration/migration.go

```diff
@@ -246,7 +255,7 @@ func (c *Controller) Execute() bool {
 
 	if err != nil {
 		log.Log.Reason(err).Infof("reenqueuing Migration %v", key)
-		c.Queue.AddRateLimited(key)
+		c.Queue.AddWithOpts(priorityqueue.AddOpts{Priority: priority}, key)
```

Comment:
Hi @vladikr @jean-edouard,
Sorry the late note. Are we sure it's okay to remove the rate-limited enqueuing?

I'm seeing a lot of logs in the ImageVolume PoC, and it seems related to how we're adding migrations to the queue:
https://storage.googleapis.com/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/13656/pull-kubevirt-e2e-k8s-1.32-sig-operator/1901289224447987712/artifacts/k8s-reporter/1/pods/1_kubevirt_virt-controller-bd647cb56-qh6xm-virt-controller.log

Reply:
Very good point... We need an `AddRateLimitedWithOpts()`. I'll look into it tomorrow, thanks for the heads-up!

Reply:
Wow! Nice catch!